src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,6 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    public abstract boolean isMutating();
```

Comment:
Need header comments to tell what this method is for. Particularly since it is abstract, need to inform developers its purpose.